stackit/internal/services/iaas/network/resource.go

```diff
@@ -141,20 +182,92 @@ func (r *networkResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 				},
 			},
 			"nameservers": schema.ListAttribute{
-				Description: "The nameservers of the network.",
+				Description:        "The nameservers of the network. This field is deprecated and will be removed soon, use `ipv4_nameservers` to configure the nameservers for IPv4.",
+				DeprecationMessage: "Use `ipv4_nameservers` to configure the nameservers for IPv4.",
+				Optional:           true,
+				Computed:           true,
+				ElementType:        types.StringType,
+			},
+			"no_ipv4_gateway": schema.BoolAttribute{
+				Description: "If set to `true`, the first IP of the network will be assigned as the IPv4 gateway.",
```

Comment:
Shouldn't this be:
```suggestion
				Description: "If set to `true`, the network doesn't have a gateway.",
```

Reply:
According to the API docs [here](https://docs.api.stackit.cloud/documentation/iaas/version/v1beta1#tag/Network/operation/v1beta1CreateNetwork)

Reply:
Same for the IPv6 one